be/src/storage/lake/lake_primary_index.cpp

```diff
@@ -53,6 +54,11 @@ bool LakePrimaryIndex::is_load(int64_t base_version) {
     return _loaded && _data_version >= base_version;
 }
 
+Status LakePrimaryIndex::insert(uint32_t rssid, const vector<uint32_t>& rowids, const Column& pks) {
+    std::lock_guard<std::mutex> lg(_mutex);
```

Comment:
Plz don't use mutex here. control multi thread insert in pk_index_loader.cpp

Reply:
If we do so, the mutex is also needed to be saved in another unordered_map which maps tablet id to mutex in pk_index_loader.cpp.